src/main/java/com/epam/training/auction_backend/services/AuctionRepository.java

```diff
@@ -0,0 +1,17 @@
+package com.epam.training.auction_backend.services;
+
+import java.util.List;
+import java.util.Optional;
+
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.transaction.annotation.Transactional;
+
+import com.epam.training.auction.common.AuctionTransferObject;
+import com.epam.training.auction_backend.entity.Auction;
+
+@Transactional
+public interface AuctionRepository extends CrudRepository<Auction, Long> {
+	public Optional<AuctionTransferObject> findOne(long id);
+	public Auction save(Auction auction);
+	public List<AuctionTransferObject> findAllByIsActive(boolean isActive);
+}
```

Comment:
I think 'All' here is redundant. Try to call it just 'findByIsActive' and it should work the same way